projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -8,13 +8,12 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "cy:run:ci:1905": "cypress run --config-file cypress.ci.1905.json --record --key $CYPRESS_KEY --tag \"1905,b2c,all\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
-    "cy:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,all\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
-    "cy:run:ci:2005:b2b": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,all\" --spec \"cypress/integration/b2b/**/*.e2e-spec.ts\"",
+    "cy:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,all,parallel\" --parallel --group $STAGE_NAME --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
+    "cy:run:ci:flaky:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,flaky\" --parallel --group flaky --spec \"cypress/integration/!(vendor|b2b)/**/*.flaky-e2e-spec.ts\"",
```

Comment:
We should add `parallel` tag to every command that is using `--parallel`

Reply:
Good catch. Will do.